app/bundles/InstallBundle/Views/Install/base.html.twig

```diff
@@ -0,0 +1,35 @@
+{#
+    @copyright   2014 Mautic Contributors. All rights reserved
+    @author      Mautic
+    @link        http://mautic.org
+    @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+#}
+
```

Comment:
Let's remove these comments. We used to have them everywhere but it's not necessary anymore.

Reply:
@escopecz I did remove this